src/biokbase/narrative/common/kbtypes.py

```diff
@@ -738,12 +767,24 @@ class ContigSet(tls.Unicode, TypeMeta):
         class v1_1(tls.Unicode, TypeMeta):
             """Type spec for the "ContigSet" object"""
             info_text = "KBaseSearch.ContigSet-1.1"
+    class Assembly(tls.Unicode, TypeMeta):
```

Comment:
KBaseSearch.Assembly is not registered.  There is a separate ContigSet and Genome defined in Search for legacy reasons having to do with the Genome and ContigSet objects defined in KBaseGenomes.  The new Assembly and GenomeAnnotation types do not need separate Search types defined.